specification/communication/data-plane/Messages/readme.csharp.md

```diff
@@ -0,0 +1,15 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
```

Comment:
Why do we have this for a data plane library?

Reply:
I am using a similar format as Chat:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/Chat/readme.csharp.md

Reply:
I'm guessing they mistakenly copied that from elsewhere too?  I don't think we want this file at all.  It's for automatically generating a C# management plane library which doesn't make sense as you've got a hand generated data plane library here.

Reply:
Okay, thank you for explaining, I will take it down.

Reply:
removed @tg-msft